CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,19 @@
 
 import Foundation
 
-print("Hello, World!")
+let myLottoNumbers: Set<Int> =  [1, 2, 3, 4, 5, 6]
+let numbers = Array(1...45)
```

Comment:
number에는 타입명시를 안해주신 이유가 있을까요?
또한 `numbers`라는 네이밍이 어떤 숫자들을 의미하는지 네이밍만 보고서는 파악하기 힘들 것 같아요. 어떻게 수정해 줄 수 있을까요?
제 의견은 통일시켜줘서 타입명시를 해주시면 좋을 것 같아요!

(+ 타입명시를 해줬을 때 이점에 대해서도 알아보시면 좋을 것 같아요.👍)

Reply:
```suggestion
var numberfull: [Int] = Array<Int>(1...45)
```
타입명시를 안했던것은 좀더 코드를 보기 간결하게?하기위해서 였습니다. 
그런데 말씀주신대로 타입명시를 타입을 명시하지 않으면 다른 타입이 설정될수 있어 타입명시를 해주는게 좋은 거같아요!
그래서 타입명시를 해주고, 네이밍을 좀더 의미있게?해줘봤습니다!